client/src/main/scala/com.gu.contentapi.client/utils/QueryStringParams.scala

```diff
@@ -5,7 +5,8 @@ import java.net.URLEncoder
 object QueryStringParams {
   def apply(parameters: Iterable[(String, String)]) = {
     def encodeParameter(p: String): String = p match {
-      case other => URLEncoder.encode(other, "UTF-8")
+      //api-gateway IAM authorisation requires that spaces are encoded as %20, not +
```

Comment:
Is this documented somewhere?

Reply:
I saw it mentioned in a guide about s3 authorised requests

Reply:
Could you please link to it?

Reply:
This doc mentions it - https://docs.aws.amazon.com/general/latest/gr/sigv4-create-canonical-request.html
"the space character must be encoded as %20 (not using '+', as some encoding schemes do)"

Reply:
Thanks. Would you mind adding this link to the comment?